### ISSUE TITLE
Class 'Jenssegers\\Date\\Date' not found

### ISSUE THREAD

**Issue by soderluk** [nordsoftware/lumen-doctrine#14]
### What did you do?

Tried to delete an entity, having soft deletes event listener on.
### What did you expect to happen?

Entity deleted_at property expected to have `Date::now()`
### What actually happened?

Fatal error was thrown.
### What versions of PHP, Lumen and this module are you using?

PHP: 5.6.23
Lumen: 5.2
lumen-doctrine: ^2.4

Please include code that reproduces the issue. The best reproductions are self-contained scripts with minimal dependencies.

``` php
$this->deleteEntityAndCommit($user);
```

Above will throw:

``` javascript
"exception": "Symfony\\Component\\Debug\\Exception\\FatalErrorException",
  "message": "Class 'Jenssegers\\Date\\Date' not found",
  "code": 1,
  "file": "vendor/nordsoftware/lumen-doctrine/src/EventListeners/SoftDeletesListener.php",
  "line": 37,
```

**Comment by Jalle19** [comment]
Just a missing dependency, but do we really want some date library just to get the current time? It seems it handles different locales etc. but I'm still not convinced.

**Comment by soderluk** [comment]
@Jalle19: This should be an easy fix, just change `Date::now()` to `Carbon::now()`

**Comment by soderluk** [comment]
This is now fixed in 2.4.1
